src/AutoBuild.jl

```diff
@@ -707,6 +707,9 @@ function autobuild(dir::AbstractString,
         # Unsymlink all the deps from the dest_prefix
         cleanup_dependencies(prefix, artifact_paths)
 
+        # Search for dead links; raise warnings about them.
+        warn_deadlinks(prefix)
```

Comment:
Should this be
```suggestion
        warn_deadlinks(prefix.path)
```
or something like that?  `prefix::Prefix`, but `warn_deadlinks` takes an `AbstractString` as input